diff --git a/chrome/app/settings_strings.grdp b/chrome/app/settings_strings.grdp
index 763bba112d55b235b1a4a11ea027342ef3920dfe..354f2ea453e5bbd89f8874109be62bc546289bdd 100644
--- a/chrome/app/settings_strings.grdp
+++ b/chrome/app/settings_strings.grdp
@@ -4487,7 +4487,7 @@
         AI innovations
       </message>
       <message name="IDS_SETTINGS_AI_PAGE_TITLE" desc="This label is on the left Navigation sidebar in Chrome’s settings. It tells the user that they can see settings related to AI (artificial intelligence) in this section. It is a noun phrase that refers to our collection of AI settings. These settings include Help me write and more.">
-        AI in Chrome
+        AI in Sponari
       </message>
       <message name="IDS_SETTINGS_AI_PAGE_MAIN_SUBLABEL_3" desc="This is a bullet point on a card that sets the context for the ‘AI innovations’ section of Chrome’s settings. Here, the user can see settings related to AI (artificial intelligence) in this section; these settings include Help me write and more. This bullet point is after the ‘Reimagine Chrome, supercharged with AI’ title and ‘Things to consider’ subheading. We want users to read this card and understand the considerations that apply to all Chrome’s AI features. This bullet point specifically notes that AI features are actively developing and results may include errors. The tone should be informative.">
         Data sent to Google may be seen by human reviewers to improve Chrome AI’s technology
