diff --git a/chrome/common/webui_url_constants.cc b/chrome/common/webui_url_constants.cc
index 4c12bc642955f1c2ff6f618e70a0ce60374afb1f..2298fa4ae6b8accdab6478515e41cf1e52ac4069 100644
--- a/chrome/common/webui_url_constants.cc
+++ b/chrome/common/webui_url_constants.cc
@@ -176,6 +176,9 @@ base::span<const base::cstring_view> ChromeURLHosts() {
       kChromeUISignInInternalsHost,
       kChromeUISiteEngagementHost,
       kChromeUISkillsHost,
+#if !BUILDFLAG(IS_ANDROID)
+      kChromeUISponariSettingsHost,
+#endif
       kChromeUISubresourceFilterInternalsHost,
 #if !BUILDFLAG(IS_ANDROID)
       kChromeUISuggestInternalsHost,
